### PR TITLE
GHA: don't install libsecp256k1 and libsodium

### PR DESCRIPTION
**Pull request by coot**
Also fix building haddocks.

### CODE REVIEW

File: .github/workflows/github-page.yml [hunk 1]
```diff
@@ -22,15 +22,8 @@ jobs:
 
     env:
       # current ref from: 27.02.2022
-      SECP256K1_REF: ac83be33d0956faf6b7f61a60ab524ef7d6a473a
-
       SECP_CACHE_VERSION: "2022-12-29"
```
Comment (jasagredo) [inline .github/workflows/github-page.yml:25]: I think you can remove also this one?

Comment (coot) [reply .github/workflows/github-page.yml:25]: yes, :)

**Review by bolt12 — APPROVED** [review]

**Comment by coot** [comment]
bors merge

**Comment by jasagredo** [comment]
This won't merge as is. Bors is expecting `9.2.5, windows-latest` as a build job. Modify `bors.toml`

**Pull request merged**